packages/fuels/src/lib.rs

```diff
@@ -75,7 +75,7 @@ pub mod prelude {
         tx::Salt,
         types::{
             bech32::{Bech32Address, Bech32ContractId},
-            errors::Error,
+            errors::{Error, Result},
```

Comment:
This is suprising to me. Are we defining our own `Result` type?

Reply:
@iqdecay Are you talking about this https://github.com/FuelLabs/fuels-rs/pull/796/files#diff-f9b98bdf9d483a3e5875ff6df383880efd6759d37e2c24cbab51d56fdc8c59f2R32?

If so, this is a fairly common and encouraged Rust pattern: https://doc.rust-lang.org/rust-by-example/error/result/result_alias.html.

Reply:
Oh I didn't know that! Then if it's recommended it will not confuse people 😊